.github/workflows/ci.yml

```
@@ -122,7 +122,7 @@ jobs:
           run-tests-ext: sh
     steps:
     - name: Checkout
-      uses: actions/checkout@v3
+      uses: actions/checkout@v4
```

1. Avoid having to change the version again (maybe?).
2. Inhibit checkout progress spam.
```suggestion
      uses: actions/checkout@master
      with:
        show-progress: false
```